frontend/lib/about/dev_settings_view.dart

```diff
@@ -30,7 +34,7 @@ final sampleActivationCodeBavaria = DynamicActivationCode(
   totpSecret: base32.decode("MZLBSF6VHD56ROVG55J6OKJCZIPVDPCX"),
 );
 
-final sampleActivationCodeNuernberg = DynamicActivationCode(
+final sampleActivationCodeNuernberg = DynamicUserCode(
   info: CardInfo(
     fullName: "Jane Doe",
```

Comment:
```suggestion
    fullName: "Erika Mustermann",
```
Closing https://github.com/digitalfabrik/entitlementcard/issues/805 :D